### ISSUE TITLE
Meta: Why forking a project instead of contributing a feature?

### ISSUE THREAD

**Issue by vsund** [hackergarten/hackadoc#36]
Hey, I'm currently in the position of setting up an own instance of HackMD/HackADoc. I thought much about which of these softwares I'd like to use and I'm yet not sure.
This raises the questions why to fork at all. After a fork you need to backport many features/fixes from the original version. So integrating this as feature in the original HackMD might be better.

Did you evaluate integrating this as feature into HackMD? I would imagine a dropdown box where I can select in which markup language I'd like to use, later $people could add more themselves.

**Comment by madmas** [comment]
HI,

we started as an experiment to enable HackMD to use AsciiDoctor instead. 

As the code base from HackMD was not built to support multiple markup languages and also the name implies the focus in MD, we decided just to go on with HackADoc for now. But it's not set to stone ;-)

So, yes we also discussed on integrating it, but it cannot be seen as  feature right now as it's touching the foundation of the software and would result in a larger refactoring.
